tests/cases/stable/py/embeddings/simple_tokenizer.py

```
@@ -0,0 +1,18 @@
+# {
+#   "Seq": [
+#     { "Depends": "py-lib-word_piece_tokenizer:test" },
+#     { "Depends": "py-genlayer:test" }
+#   ]
+# }
+
+from genlayer import *
```

Ruff (0.12.2)

`from genlayer import *` used; unable to detect undefined names

(F403)